src/hooks/useSelectedRevisions.ts

```diff
@@ -72,8 +76,20 @@ const useSelectRevision = () => {
 
   const deleteSelectedRevisions = (revision: RevisionsList) => {
     const newSelected = [...selectedRevisions];
+    if (newSelected[0].id === revision.id) {
+      //this is a base revision deletion
+      dispatch(
+        deleteRevision({
+          selectedRevisions: newSelected,
```

Comment:
see below: I think it's more correct to call `slice` here. The code is then more similar to the code for deleting a new revision: change the selected revisions first here, and dispatch the resulting array then.

Reply:
Yes, agree, best to keep things consistent.